pulp_ansible/app/viewsets.py

```diff
@@ -554,6 +555,13 @@ class AnsibleRepositoryViewSet(RepositoryViewSet, ModifyRepositoryActionMixin, R
     queryset = AnsibleRepository.objects.all()
     serializer_class = AnsibleRepositorySerializer
 
+    def get_queryset(self):
+        qs = super().get_queryset()
+        if getattr(self, "action", "") == "list":
```

Comment:
When doing the read (single object), do we still want to see the task?

Reply:
oops... good catch :+1: . Yes, UI uses it in detail as well.